src/bad_words/key.json

```diff
@@ -0,0 +1,27 @@
+{
+    "en": 
```

Comment:
we need a function to add more bad words

Reply:
Maybe an `extraBadWords` prop on `ChatProvider` component, and move `isFilterBadWords` prop on `ChatScreen` to `ChatProvider` component too

Reply:
I tried to add isFilterBadWords in ChatProvider but it's redundant. We just need pass an array `blackListWords` and then enable filter black list

Reply:
Then we won't need the prop `isFilterBadWords` we only need to config an `blackListWords` prop on ChatProvider is enough, if we need to filter we only have to add the list there
PS. This is same with my comment below